CMakeLists.txt

```diff
@@ -137,16 +187,17 @@ target_compile_options(
                           ${BUILD_TYPE_COMPILE_FLAGS})
 target_link_options(${PROJECT_NAME} PRIVATE ${BUILD_TYPE_LINK_FLAGS})
 target_include_directories(${PROJECT_NAME} PRIVATE ${CMAKE_CURRENT_BINARY_DIR})
-target_link_libraries(${PROJECT_NAME} PRIVATE ${LIBRARY_NAME} ${TEST_LIBRARIES})
+target_link_libraries(${PROJECT_NAME} PRIVATE ${NESO_LIBRARY_NAME}
+                                              ${TEST_LIBRARIES})
 add_sycl_to_target(TARGET ${PROJECT_NAME} SOURCES ${MAIN_SRC})
 
 # find MPI
 find_package(MPI REQUIRED)
```

Comment:
Better to find these packages up at the top of the file, along with the rest. They can then be linked to the library with the rest, around line 174, rather than separately here.

Reply:
I don't follow - I changed the neso library name not the cmake structure.

Reply:
@cmacmackin

Reply:
Think this comment belonged somewhere else. It's not an important issue anyway, so will mark this as resolved.